ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/DefaultPrivacyController.java

```diff
@@ -245,7 +246,8 @@ public long determineBesuNonce(
   }
 
   @Override
-  public PrivacyGroup retrievePrivacyGroup(final String privacyGroupId, final String enclaveKey) {
+  public PrivacyGroup retrieveOffChainPrivacyGroup(
```

Comment:
Can we change this method to return an Optional?

Reply:
done